rinex/src/lib.rs

```diff
@@ -325,15 +276,15 @@ impl Rinex {
     }
 
     /// Converts self to CRINEX1 compressed format,
-    /// whatever the RINEX revision might be.
+    /// whatever the RINEX revision might be.  
```

Comment:
```suggestion
    /// whatever the RINEX revision might be.
```

Reply:
This is how to insert a new line in the documentation, makes it more readable IMO

Reply:
Oh, I had no idea. I use an empty line (and configure my editor to trim on save, so that would be a surprise).